broadcast/views/auth/login.mako

```diff
@@ -0,0 +1,8 @@
+<%inherit file="/_inner.mako"/>
+<%namespace name="login" file="_login.mako"/>
+
+<h1>${_('Log in')}</h1>
+
+<div id="login-form" class="login-form">
```

Comment:
Should use `<section>` here, because the skin updates contain code that will apply uniform styling to any <section> element found in the body.default